README.md

```diff
@@ -321,6 +321,15 @@ Example:
 
 ```
 
+
+#### Using Grafana in a reverse proxy environment
+
+When defining `grafana_dashboard` and / or `grafana_datasource` you have to make sure you
+set `grafana_api_path` according to your needs. By default this value is set to `/api`, if
```

Comment:
If '/api' is set by default, you wouldn't be required to set the grafana_api_path unless you're using sub-paths. I would re-phrase this to make it clear that it's not a required parameter unless you are changing it from the default setting.